oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/menu/UIMenuManagementService.java

```diff
@@ -48,7 +49,7 @@ public class UIMenuManagementService implements Service, Runnable {
     private ModuleManager moduleManager;
     private CompletableFuture<List<MenuItem>> menuItemFuture;
```

Comment:
Future is going to be changed through another PR, right?

Reply:
Yes.